client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -678,9 +678,18 @@ export default {
 
     save () {
       const comments = this.segment.relationships.comments ? { ...this.segment.relationships.comments } : null
+      const rel = this.updateRelationships()
+
+      const payload = {
+        data: {
+          id: this.segment.id,
+          type: 'StatementSegment',
+          relationships: { assignee: rel.assignee, place: rel.place }
+        }
+      }
```

Comment:
nitpick; indentation may need to be fixed in my suggestion

```suggestion
      const { assignee, place } = this.updateRelationships()

      const payload = {
        data: {
          id: this.segment.id,
          type: 'StatementSegment',
          relationships: {
            assignee,
            place
            }
        }
      }
```

Reply:
I did it like that in the beginning, but for some reason sonarCloud complained about it. Now its fine . Don't get it.
0c1aec2fd9c492da1f7e43eee516588b545fafca